### PR TITLE
Updated documentation for pandas.DataFrame.map

### PR DESCRIPTION
**Pull request by DarthKitten2130**
closes issue #54648 by adding another example of the map method using a standard func, along with additional keyword arguments.

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -10088,6 +10088,20 @@ def map(
         0  3  4
         1  5  5
 
+        >>> def func(x,y): # x refers to the element in the DataFrame
```
Comment (mroeschke) [inline pandas/core/frame.py:10091]: Can you just define a lambda that takes an extra argument?

Comment (DarthKitten2130) [reply pandas/core/frame.py:10091]: Actually my goal was to demonstrate how the map function can be used with standard functions that are not lambdas. If you think I should use a more complex example to show that, I could do so.

Comment (mroeschke) [reply pandas/core/frame.py:10091]: Can you just provide a built in function like `round`?

Comment (DarthKitten2130) [reply pandas/core/frame.py]: Thank you for the advice! I did just that, you can have a look.

**Comment by mroeschke** [comment]
Thanks for the pull request, but it appears to have gone stale. If interested in continuing, please merge in the main branch, address any review comments and/or failing tests, and we can reopen.

**Pull request closed without merging**